Add composite indexes support to CosmosDB SQL container (#8792)
terraform-providers/terraform-provider-azurerm: website/docs/r/cosmosdb_sql_container.html.markdown

* `path` - Path that is excluded from indexing.

A `composite_index` block supports the following:

* `index` - One or more `index` blocks as defined below.

An `index` block supports the following:

* `path` - Path for which the indexing behavior applies to.

> * `order` - Order of the index. Can be either `ascending` or `descending`.

Reviewer: these are cased "Ascending" and "Descending" in the SDK - so can we make this:

```suggestion
* `order` - Order of the index. Possible values are `Ascending` or `Descending`.
```

(which also changes the text slightly to match other resources)
